tests/integration/test_multi_objective_bayesian_optimization.py

```diff
@@ -72,6 +73,16 @@
             -3.2095,
             id="BatchMonteCarloExpectedHypervolumeImprovement/4",
         ),
+        pytest.param(
+            10,
+            EfficientGlobalOptimization(
+                HippoPenalizationAcquisitionFunction(),
+                num_query_points=4,
+                optimizer=generate_continuous_optimizer(num_initial_samples=500),
+            ),
+            -3.2095,
+            id="qehvi_vlmop2_q_4",
```

Comment:
give it a better id>?

Reply:
will do! although not sure what these ids are supposed to do

Reply:
@uri-granta ?

Reply:
fixed now